src/config/color.py

```diff
@@ -1,56 +1,75 @@
+import zipfile
 from pathlib import Path
 
 import torch
+from fastai.vision.models import resnet18
+from fastai.vision.models.unet import DynamicUnet
+from loguru import logger
 from src.data.color import CustomDataset
```

Comment:
I suddenly understood that we have kind a confusion in naming. I suppose this custom dataset should be called `ColorDataset` or smth like this. (Same as we have `BaselineDataset` in baseline branch and `SimpleDataset` for template objects.)

What do you think about it? If you agree I will simply change it in my branch (colorization-plus-gan).

Reply:
Yes it makes sense to have different names if they do different things